### PR TITLE
Update references to "master" in the main branch.

### PR DESCRIPTION
**Pull request by dkinzer**
Updates some links and references to "master" for the new main branch.

### CODE REVIEW

File: app/assets/javascripts/blacklight/blacklight.js [hunk 1]
```diff
@@ -371,7 +371,7 @@ Blacklight.modal.onFailure = function (jqXHR, textStatus, errorThrown) {
 Blacklight.modal.receiveAjax = function (contents) {
   // does it have a data- selector for container?
   // important we don't execute script tags, we shouldn't.
-  // code modelled off of JQuery ajax.load. https://github.com/jquery/jquery/blob/master/src/ajax/load.js?source=c#L62
+  // code modelled off of JQuery ajax.load. https://github.com/jquery/jquery/blob/main/src/ajax/load.js?source=c#L62
```
Comment (jcoyne) [inline app/assets/javascripts/blacklight/blacklight.js:374]: This is not our branch

File: app/javascript/blacklight/modal.js [hunk 2]
```diff
@@ -120,7 +120,7 @@ Blacklight.modal.onFailure = function(jqXHR, textStatus, errorThrown) {
 Blacklight.modal.receiveAjax = function (contents) {
     // does it have a data- selector for container?
     // important we don't execute script tags, we shouldn't.
-    // code modelled off of JQuery ajax.load. https://github.com/jquery/jquery/blob/master/src/ajax/load.js?source=c#L62
+    // code modelled off of JQuery ajax.load. https://github.com/jquery/jquery/blob/main/src/ajax/load.js?source=c#L62
```
Comment (jcoyne) [inline app/javascript/blacklight/modal.js:123]: This is not our branch

**Review by jcoyne — CHANGES_REQUESTED** [review]

File: app/assets/javascripts/blacklight/blacklight.js (hunk 1 above)
Comment (cbeer) [reply app/assets/javascripts/blacklight/blacklight.js:374]: They're using `main` now too 🎉

File: app/javascript/blacklight/modal.js (hunk 2 above)
Comment (jcoyne) [reply app/javascript/blacklight/modal.js:123]: Hmm, looks like they made this change too.

**Review by jcoyne — APPROVED** [review]

**Review by bess — APPROVED** [review]

**Pull request merged**